src/dotnet-affected/Commands/Context/CommandExecutionContext.cs

```diff
@@ -61,19 +62,17 @@ protected virtual void Dispose(bool disposing)
 
         /// <summary>
         /// Builds a <see cref="ProjectGraph"/> from all found project files
-        /// inside the <see cref="CommandExecutionData.RepositoryPath"/>.
+        /// inside the <see cref="CommandExecutionData.SolutionPath"/> or <see cref="CommandExecutionData.RepositoryPath"/>.
         /// </summary>
         /// <returns>A new Project Graph.</returns>
         private ProjectGraph BuildProjectGraph()
         {
             // Find all csproj and build the dependency tree
             this.WriteLine($"Finding all csproj at {this.executionData.RepositoryPath}");
```

Comment:
chore: move this log to `FindProjectsInDirectory` and add a corresponding one inside `FindProjectsInSolution`